### ISSUE TITLE
Missing Tiles

### ISSUE THREAD

**Issue by fabm3n** [the1killer/sm_overview#7]
Here are some missing tiles:
Missing tile at 14,7 for id 1085201 NONE
Missing tile at 13,14 for id 1085201 NONE
Missing tile at -10,12 for id 1004601 NONE

Map Seed: 46380146

**Comment by the1killer** [comment]
I will try to write up a guide on how to generate the tiles for yourself but here is brief synopsis:
Have to get code I wrote for teleporting, and you teleport above the tile at its center at like 400-600 height and then take a screenshot on the way down, but then have to open in an image tool and crop it to 500x500 centered, then place in the **`assets/img/`** folder with file name **`$TILEID$.jpg`**, so in the OP's case **`1085201.jpg`**. 

Took me some trial and error, especially on the cliff tiles which will be many of the missing tiles.

You can get the **/cell** and **/tp** commands from [SurvivalGame.lua](https://github.com/the1killer/sm_overview/blob/main/SurvivalGame.lua) which is probably out of date with the current game's file so don't just overwrite.

Then you can do **/cell 14,7,500** to teleport to that cell's center at 500 height, I would usually let it fall for a second or so then snap the screenshot. Also if you are not nearby this cell then you may have to teleport there first and let it load, then teleport above and screenshot.

**Comment by gamingoninsulin** [comment]
yes i can say for sure the /cell and /tp are outdated do you have an updated /cell and /tp command or have the bits to get it to work again

**Comment by the1killer** [comment]
The commands should still work, but you have to copy the code into different locations in the file most likely. For the chat commands there's two locations, one to register the command and another for the handler when the command is called, you should be able to copy them from the files in the repo and paste just those sections into the current game SurvivalGame.lua

**Comment by gamingoninsulin** [comment]
i am trying to get this to work

link to to my SurvivalGame.lua:
https://gist.github.com/gamingoninsulin/dd76aee46a1f0a24e45cf10cd1185ebc

i dont know much about lua so i also dont know what i am doing wrong?!?
if you have the time can you help me what i m doing wrong

i dont know where to look for i tried copy paste in same functions in the same place but thiss not work
or mabe if you have no time but still willing to help point me into the direction of the right functions.

kind regard Insulin.

**Comment by the1killer** [comment]
Your script is working in game for me? if it isn't can you relay what it says in chat console? or game logs? if it has a lua error should be in game log.

to use it you press enter for chat then can do things like:
```
/tp here
/cell here
/tp -1692,-1697,10
/cell -27,-27,500
```
The first two tell you your current coordinates. tp is exact coords, cell is cell number

The last one is what i used a lot to take pictures of different tiles, it teleports you to the center of the cell/tile and a specified height, i can't remember what height i used a lot to get them to properly match up with other tiles, i would usually teleport up height, let it fall for a few seconds, take a screenshot, then crop it square at say 300x300 or 500x500 i forget.
